uk_geo_utils/management/commands/import_onspd.py

```diff
@@ -27,26 +27,16 @@
 }
 
 
-class Command(BaseCommand):
+class Command(BaseUpdater):
     """
     To import ONSPD, grab the latest release:
     https://ons.maps.arcgis.com/home/search.html?t=content&q=ONS%20Postcode%20Directory
-    and run
-    python manage.py import_onspd /path/to/data
+    and run:
+        python manage.py update_onspd --data-path /home/will/Downloads/ONSPD_MAY_2024/Data
```

Comment:
In fairness, it is quite a good guess that the user of this package will be called Will :laughing: